NLibrary/Classes/Service Files/FirebaseService.swift

```diff
@@ -23,6 +27,9 @@ public enum APIRequestResult: Error {
     private var repoDashBoard: DashboardRepoProtocol?
     private var repoPlaylist: PlaylistRepo?
 
+    public required override init() {
+        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5382838)